examples/src/App.tsx

```diff
@@ -39,3 +50,17 @@ export const App = () => {
     </FluentProvider>
   );
 };
+
+const routeElements: Record<TRoute, JSX.Element> = {
+  [routes.mainMenu]: <MainMenuPage />,
+  [routes.Home]: <WelcomePage />,
+  [routes.Theme]: <ThemePage />,
+  [routes.IconCatalog]: <IconPage />,
+  [routes.Stepper]: <StepperPage />,
+  [routes.Slider]: <SliderPage />,
+  [routes.PasswordInput]: <PasswordInputPage />,
+  [routes.EmptyView]: <EmptyViewPage />,
+  [routes.TableUtilities]: <TableUtilitiesPage />,
+  [routes.TabListUtilities]: <FluentUiTabStylesPage />,
+  [routes.Illustrations]: <IllustrationPage />,
+};
```

Comment:
Separates data from spec to avoid circular dependencies. If specification and elements are combined, the elements should not in turn depend on the specification (but they do). This caused a problem with vite's HMR, so the list is now here (I didn't bother creating a separate file for this as it's only used here).